YOLayout/Box/YOHBox.m

```diff
@@ -0,0 +1,41 @@
+//
+//  YOHBox.m
+//  YOLayoutExample
+//
+//  Created by Gabriel on 3/13/15.
+//  Copyright (c) 2015 YOLayout. All rights reserved.
+//
+
+#import "YOHBox.h"
+
+@implementation YOHBox
+
+- (void)viewInit {
+  [super viewInit];
+
+  YOSelf yself = self;
+#if TARGET_OS_IPHONE
+  self.layout = [YOLayout layoutWithLayoutBlock:^(id<YOLayout> layout, CGSize size) {
```

Comment:
Cool.. I made viewLayout the property.
I deprecated layout (points to viewLayout) and updated docs.

Similarly on Cocoa side, I made YONSView -> YOView. Now stuff is unified.